res/openvpn-gui-res-no.rc

```diff
@@ -371,29 +370,29 @@ BEGIN
     /* service */
     IDS_ERR_OPEN_SCMGR_ADMIN "En feil oppsto under åpning av Service Manager. Du har sannsynligvis ikke rettigheter til å starte/stoppe en service."
-    IDS_ERR_OPEN_VPN_SERVICE "Kunnne ike åpne ""OpenVPNService"""
+    IDS_ERR_OPEN_VPN_SERVICE "Kunnne ikke åpne ""OpenVPNService"""
     IDS_ERR_START_SERVICE "En feil oppsto under oppstart av OpenVPN Service"
-    IDS_ERR_QUERY_SERVICE "En feil oppsto når status på OpenVPN Service skulle kontrolleres."
+    IDS_ERR_QUERY_SERVICE "En feil oppsto da status på OpenVPN Service skulle kontrolleres."
     IDS_ERR_SERVICE_START_FAILED "OpenVPN Service kunne ikke starte."
     IDS_ERR_OPEN_SCMGR "OpenSCManager feilet (%d)"
     IDS_ERR_STOP_SERVICE "En feil oppsto ved stopp av OpenVPN Service"
     IDS_NFO_RESTARTED "OpenVPN Service omstart utført."
 
     /* registry */
     IDS_ERR_GET_WINDOWS_DIR "Kunne ikke finne katalog hvor Windows er installert."
-    IDS_ERR_GET_PROGRAM_DIR "Kunne ikke finne path til din ""Program"" katalog."
+    IDS_ERR_GET_PROGRAM_DIR "Kunne ikke finne filstien til din ""Program"" katalog."
     IDS_ERR_OPEN_REGISTRY "Feil under åpning av register for lesning (HKLM\\SOFTWARE\\OpenVPN).\n OpenVPN er kanskje ikke installert."
-    IDS_ERR_READING_REGISTRY "Feil under lesning av register-nøkkel (HKLM\\SOFTWARE\\OpenVPN)."
-    IDS_ERR_PASSPHRASE_ATTEMPTS "Register-verdi ""passphrase_attempts"" må være ett tall mellom 1 og 9."
-    IDS_ERR_CONN_SCRIPT_TIMEOUT "Register-verdi ""connectscript_timeout"" må være ett tall mellom 0 og 99."
-    IDS_ERR_DISCONN_SCRIPT_TIMEOUT "Register-verdi ""disconnectscript_timeout"" må være ett tall mellom 1 og 99."
-    IDS_ERR_PRECONN_SCRIPT_TIMEOUT "Register-verdi ""preconnectscript_timeout"" må være ett tall mellom 1 og 99."
-    IDS_ERR_CREATE_REG_KEY "Feil under opprettelse av register-nøkkel HKLM\\SOFTWARE\\OpenVPN-GUI."
-    IDS_ERR_OPEN_WRITE_REG "Feil under åpning av register for skrivning. Du må starte programmet en gang som administrator for å oppdatere registret."
-    IDS_ERR_READ_SET_KEY "Feil under lesning og skrivning av register-verdi ""%s""."
-    IDS_ERR_WRITE_REGVALUE "Feil under skrivning av register-verdi ""HKEY_CURRENT_USER\\%s\\%s""."
+    IDS_ERR_READING_REGISTRY "Feil under lesing av registernøkkel (HKLM\\SOFTWARE\\OpenVPN)."
+    IDS_ERR_PASSPHRASE_ATTEMPTS "Registerverdi ""passphrase_attempts"" må være ett tall mellom 1 og 9."
+    IDS_ERR_CONN_SCRIPT_TIMEOUT "Registerverdi ""connectscript_timeout"" må være ett tall mellom 0 og 99."
+    IDS_ERR_DISCONN_SCRIPT_TIMEOUT "Registerverdi ""disconnectscript_timeout"" må være ett tall mellom 1 og 99."
+    IDS_ERR_PRECONN_SCRIPT_TIMEOUT "Registerverdi ""preconnectscript_timeout"" må være ett tall mellom 1 og 99."
+    IDS_ERR_CREATE_REG_KEY "Feil under opprettelse av registernøkkel HKLM\\SOFTWARE\\OpenVPN-GUI."
+    IDS_ERR_OPEN_WRITE_REG "Feil under åpning av register for skriving. Du må starte programmet en gang som administrator for å oppdatere registret."
```

Comment:
I would rather say:  "Kunne ikke oppdatere registeret.  Du må starte programmet med administator-rettigheter for å kunne oppdatere innstillinger i registeret."